api/v1alpha1/project_config_types.go

```diff
@@ -481,8 +481,8 @@ type IndexSelectorRequirement struct {
 type IndexSelectorRequirementOperator string
 
 const (
-	IndexSelectorRequirementOperatorEqual    IndexSelectorRequirementOperator = "Equal"
-	IndexSelectorRequirementOperatorNotEqual IndexSelectorRequirementOperator = "NotEqual"
+	IndexSelectorRequirementOperatorEqual    IndexSelectorRequirementOperator = "Equals"
+	IndexSelectorRequirementOperatorNotEqual IndexSelectorRequirementOperator = "NotEquals"
```

Comment:
Nit: If we followed k8s' own precedent for implementation, these would be this:

```suggestion
	IndexSelectorOperatorEqual    IndexSelectorOperator = "Equals"
	IndexSelectorOperatorNotEqual IndexSelectorOperator = "NotEquals"
```

Take it or leave it. Merely a discrepancy I noticed.